mozilla-release/mobile/android/base/java/org/mozilla/gecko/BrowserApp.java

```diff
@@ -3119,7 +3119,7 @@ public void onPropertyAnimationEnd() {
     private void hideWebContent() {
         // The view is set to INVISIBLE, rather than GONE, to avoid
         // the additional requestLayout() call.
-        mLayerView.setVisibility(View.INVISIBLE);
+        // !!! Cliqz !!! mLayerView.setVisibility(View.INVISIBLE);
```

Comment:
@spacifici would suggest to have a consistent way of handling this type of changes. Here you had a comment on the same line, later its one line above.

My suggestion would be something like:
/* Cliqz start*/
//whatever new code or change or comment
/* Cliqz end */

would be easier to merge in the future.

@alver-cliqz could suggest something even better

Reply:
There are one simple rules - try not mixing our comments with original code in one line, because it will be hard later to merge them. You can use different way to comment a block of code: block comment (/* .. */), #if 0 ... #endif (file must be preprocessed in jar.mn), etc.

In desktop we prefer to use #if

Reply:
@alver-cliqz, using `#if 0` is not possible in the java code, I'll opt for the `/* Cliqz start/end */` approach